WalletWasabi.Fluent/Views/Settings/GeneralSettingsTabView.axaml

```diff
@@ -16,27 +16,27 @@
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Start Wasabi with the system" />
+      <TextBlock Text="Run Wasabi when computer starts" />
       <ToggleSwitch IsChecked="{Binding RunOnSystemStartup}" Command="{Binding StartupCommand}" />
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Hide Window on close" />
+      <TextBlock Text="Minimize Wasabi on Close" />
```

Comment:
Can you explain why `Close` has capital C?

I don't really understand when use capital letters and when not. It still seems random to me. Can you explain what rules you follow in this regard?

Reply:
I used capital letters on words that important to catch the user eyes.

Reply:
How about simply use normal capitalization. I mean not emphasizing any words.